### PR TITLE
Set rebase_mode unsafe for some storage_vm_migration cases

### PR DESCRIPTION
**Pull request by aliang123**
blockdev_snapshot:set rebase_mode unsafe
blockdev_commit:set rebase_mode unsafe
blockdev_stream:set rebase_mode unsafe
blockdev_full_mirror:set rebase_mode unsafe
blockdev_full_backup_multi_disks:set rebase mode unsafe

Signed-off-by: Aihua Liang <aliang@redhat.com>

id:1817908

### CODE REVIEW

**Comment by aliang123** [comment]
(01/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_full_mirror.simple_test.with_data_plane.q35: PASS (113.06 s)
 (02/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_full_mirror.simple_test.q35: PASS (119.74 s)
 (03/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_full_mirror.simple_test.dst_cluster_size_512.with_data_plane.q35: PASS (135.74 s)
 (04/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_full_mirror.simple_test.dst_cluster_size_512.q35: PASS (115.29 s)
 (05/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_full_mirror.simple_test.dst_cluster_size_2M.with_data_plane.q35: PASS (116.36 s)
 (06/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_full_mirror.simple_test.dst_cluster_size_2M.q35: PASS (116.33 s)
 (07/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_full_mirror.simple_test.dst_random_cluster_size.with_data_plane.q35: PASS (117.38 s)
 (08/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_full_mirror.simple_test.dst_random_cluster_size.q35: PASS (115.41 s)
 (09/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_snapshot.q35: PASS (104.52 s)
 (10/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_snapshot.dst_cluster_size_512.q35: PASS (97.40 s)
 (11/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_snapshot.dst_cluster_size_2M.q35: PASS (97.28 s)
 (12/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_commit.q35: PASS (97.43 s)
 (13/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_stream.q35: PASS (98.30 s)
 (14/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_multi_disks.with_data_plane.q35: PASS (112.96 s)
 (15/15) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_multi_disks.q35: PASS (135.38 s)

**Comment by ngu-niny** [comment]
@aliang123 Aihua, seems only the 'blockdev_snapshot' case need the unsafe rebase_mode.

**Comment by aliang123** [comment]
@ngu-niny, all test cases that do qemu-img rebase all needs rebase_mode=unsafe.

**Comment by ngu-niny** [comment]
Talked with @aliang123, the unsafe rebase_mode might be used in future cases, so we can keep it.

LGTM.

Acked-by: Nini Gu <ngu@redhat.com>

File: qemu/tests/cfg/blockdev_commit.cfg [hunk 1]
```diff
@@ -33,6 +33,7 @@
     #mode = "absolute-paths"
     base_tag = sn1
     top_tag = sn3
+    rebase_mode = unsafe
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_commit.cfg]: There are two different modes in which "rebase" can operate: safe mode and unsafe mode, if  rebase_mode is defined in global scope in cfg files, 'unsafe' will always be used for rebase operation, so, why not using variants here, which can let rebase choose safe mode or unsafe mode?

Comment (aliang123) [reply qemu/tests/cfg/blockdev_commit.cfg]: In all these test cases, only unsafe mode is needed for qemu-img rebase.

**Comment by zhencliu** [comment]
LGTM

**Pull request merged**